pkg/controllers/nodeclass/status/subnet.go

```diff
@@ -56,5 +56,5 @@ func (s *Subnet) Reconcile(ctx context.Context, nodeClass *v1.EC2NodeClass) (rec
 		}
 	})
 	nodeClass.StatusConditions().SetTrue(v1.ConditionTypeSubnetsReady)
-	return reconcile.Result{RequeueAfter: time.Minute}, nil
+	return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
```

Comment:
Why not keep this refresh timeframe at a minute? I believe we discussed this with @engedaam. I believe if anything we wanted to decrease our retry on security groups to a minute

Reply:
reverted